packages/terra-dropdown-button/src/SplitButton.jsx

```diff
@@ -63,21 +74,50 @@ class SplitButton extends React.Component {
     super(props);
 
     this.handleDropdownButtonClick = this.handleDropdownButtonClick.bind(this);
+    this.handlePrimaryButtonClick = this.handlePrimaryButtonClick.bind(this);
     this.handleDropdownRequestClose = this.handleDropdownRequestClose.bind(this);
     this.handlePrimaryKeyDown = this.handlePrimaryKeyDown.bind(this);
     this.handlePrimaryKeyUp = this.handlePrimaryKeyUp.bind(this);
     this.handleCaretKeyDown = this.handleCaretKeyDown.bind(this);
     this.handleCaretKeyUp = this.handleCaretKeyUp.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.setListNode = this.setListNode.bind(this);
+
+    this.state = {
+      isOpen: false, caretIsActive: false, primaryIsActive: false, isKeyboardEvent: false,
```

Comment:
Same comment as above. Please change `isKeyboardEvent` to what it actually signifies w.r.t. the component.

Reply:
changed to openedViaKeyboard